ribs/emitters/_evolution_strategy_emitter.py

```diff
@@ -33,11 +33,12 @@ class EvolutionStrategyEmitter(EmitterBase):
             lambda function) that takes in no parameters and returns an instance
             of :class:`RankerBase`, or it may be a full or abbreviated ranker
             name as described in :meth:`ribs.emitters.rankers.get_ranker`.
-        restart_rule ("no_improvement" or "basic"): Method to use when checking
-            for restarts. With "basic", only the default CMA-ES convergence
-            rules will be used, while with "no_improvement", the emitter will
-            restart when none of the proposed solutions were added to the
-            archive.
+        restart_rule (int, "no_improvement", and "basic"): Method to use when
+            checking for restarts. If given an integer, then the emitter will
+            restart after this many iterations. With "basic", only the default
```

Comment:
Clarify that each iteration is a call to tell(). Also say this in the docstring for `itrs`